ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -17,21 +16,35 @@
  * Created by nata on 24.06.15.
  */
 public class AddFormTest extends TestBase {
+    private static final By NEW_HTML_FORM = By.linkText("htmlForm.form");
```

Comment:
this would fail, you can use something like `By.cssSelector("input[href='htmlForm.form']")`

Reply:
Resolved trhanks